app/assets/stylesheets/_main.scss

```diff
@@ -402,6 +403,16 @@ body {
   font-family: $govuk-font-family;
 }
 
+.govuk-form-group--error {
+  .autocomplete__input {
+    border-color: $govuk-error-colour;
+
+    &:focus {
+      border-color: $govuk-input-border-colour;
+    }
+  }
+}
+
```

Comment:
This makes sure that autocomplete inputs have a red border as class on the input is lost